.github/workflows/release.yml

```diff
@@ -37,31 +37,28 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
-          fetch-depth: 0
-      - name: SemVer
-        id: semver
-        uses: paulhatch/semantic-version@v4.0.2
+          persist-credentials: false
+      - uses: actions/setup-java@v3
         with:
-          tag_prefix: "v"
-          bump_each_commit: false
-          format: "${major}.${minor}.${patch}"
-      - name: Generate Code
-        shell: bash
-        run: ./scripts/generate.sh -v ${{ steps.semver.outputs.version }}
-      - name: Create commits
-        uses: stefanzweifel/git-auto-commit-action@v4
+          distribution: 'zulu' # See 'Supported distributions' for available options
+          java-version: '11'
+
+      - uses: shivammathur/setup-php@v2
         with:
-          commit_message: Bump version to ${{ steps.semver.outputs.version }}
-          skip_fetch: true
-      - name: Create Release
-        id: release
-        uses: actions/create-release@v1
+          php-version: "${{ matrix.php_version }}"
```

Comment:
Are you sure about matrix for release?

Reply:
Should I only test on single version or remove the test altogether?

Reply:
It is a release workflow and looks like there are no tests run here

Reply:
I removed test job from release, and used php version 7.3 as single version